### PR TITLE
Improved documentation for Xcode Project Setup

### LINKED ISSUE #455: documentation error
> When you create a new project in Xcode 7, a unit test target is included by default

This is not true if you are creating a Command Line Tool project. Furthermore if you are working on a command line tool there is no `"Defines Modules" line` to set to "Yes" as instructed.

### PR DESCRIPTION
**Pull request by masukomi**
fixes #455

### CODE REVIEW

File: Documentation/SettingUpYourXcodeProject.md [hunk 1]
```diff
@@ -1,16 +1,20 @@
 # Setting Up Tests in Your Xcode Project
 
 When you create a new project in Xcode 7, a unit test target is included
-by default. To write unit tests, you'll need to be able to use your main
-target's code from within your test target.
+by default (unless it's a Command Line Tool project). To write unit tests,
```
Comment (modocache) [inline Documentation/SettingUpYourXcodeProject.md]: Could we provide instructions on how to add a unit test target for command line tool projects?

**Comment by istx25** [comment]
>Could/should we provide instructions on how to add a unit test target for command line tool projects?

Hi @masukomi. Would you be able to update your PR to reflect @modocache's feedback.

File: Documentation/SettingUpYourXcodeProject.md [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
 ## Testing Swift Code Using Swift
 
 In order to test code written in Swift, you'll need to do two things:
 
-1. Set "defines module" in your `.xcodeproj` to `YES`.
+1. Set "Defines Module" in your `.xcodeproj` to `YES`.
 
-  * To do this in Xcode: Choose your project, then "Build Settings" header, then "Defines Modules" line, then select "Yes".
+  * To do this in Xcode: Choose your project, then "Packaging" header, 
+    then "Defines Modules" line, then select "Yes". Note: you may have
```
Comment (rastersize) [inline Documentation/SettingUpYourXcodeProject.md]: Could we apply the following change to match the settings name in Xcode:
```diff
- "Defines Modules"
+ "Defines Module"
```

I.e. drop the trailing `s` from “Modules”

**Comment by istx25** [comment]
Mind dealing with that and the merge conflicts? Cheers!

**Comment by istx25** [comment]
I'll fix the merge conflict so we can get this PR merged in.

**Comment by istx25** [comment]
Continued over at #630.

**Pull request closed without merging**